### ISSUE TITLE
How to get back the old UI? (or V3 missing features)

### ISSUE THREAD

**Issue by ivnnv** [appbaseio/dejaVu#263]
Hi,

Theres a url parameter or something to get back to the old local dejavu (im using chrome extension)?
(not appbase's because my ES is not https so thats not working because mixed content).
 
Im trying to achieve this because the new UI is missing some important v2 features.

EX: Where the select row checkboxes in editing view went?

![dv2](https://user-images.githubusercontent.com/23552631/50652569-307cd600-0f87-11e9-8e2a-a84b67cb201d.png)

vs

![dv3](https://user-images.githubusercontent.com/23552631/50652577-34105d00-0f87-11e9-8aa5-e31747c9abff.png)

Plus string values are not showing in the cell, but shown as a {...} json-like object which is kind of confusing and difficult to read (need to hover to get the actual string value).

Im very thankful for this extension, using it for quite a long time now, it makes my life easier, but theres still a lot of room for improvement in this new version.

**Comment by metagrover** [comment]
Hi @ivnnv, you can access the old dejavu at: https://opensource.appbase.io/dejavu/

We are iteratively working on the new version to get as much parity of features as possible, and will most likely get to the above enhancements once we have some critical things out of the way.

Thanks 🐱

**Comment by lakhansamani** [comment]
Hi @ivnnv 
Also, the new version of Dejavu has, select feature, its same UX as the older version, i.e. you can hover index number in the ID field and you will be able to see checkbox in editing mode

<img width="1440" alt="screenshot 2019-01-03 23 25 45" src="https://user-images.githubusercontent.com/6964334/50653355-3e465180-0faf-11e9-89b0-fb768e1d3941.png">

Regarding string showing up as `JSON` object, can you share mapping of that field, it will help us fix UX issue

Thanks

**Comment by ivnnv** [comment]
@metagrover i have double checked, and definitely theres no checkboxes showing on the row when entering editing mode :(
Your screenshot is from appbase's app, so maybe you have a working version there but not yet updated  chrome extension? idk.
I cant find a checkbox even inspecting the dom for it on the first cell of the row (neither the "select-all" check on the header)
![checkbox missing](https://user-images.githubusercontent.com/23552631/50655878-c3227280-0f91-11e9-89a7-24ff92a31471.png)


@lakhansamani heres the kind of string mapping im using in case it helps you debugging:
![string mapping 1](https://user-images.githubusercontent.com/23552631/50656041-3d52f700-0f92-11e9-9350-9a365aff1776.png)

**Comment by siddharthlatest** [comment]
@ivnnv I checked on the chrome extension, we do support showing the checkbox on hovering `Editing` mode. Do you see any console errors when it doesn't show up? That could help debug.

We were able to find the issue with the `keyword` mapping display, this will be fixed in the next release.

**Comment by ivnnv** [comment]
yep @siddharthlatest, I have an error from REACTIVESEARCH in the console, here is it:
```
vendors~main.0d71f224.js:67 REACTIVESEARCH - An error occured while parsing the URL state. SyntaxError: Unexpected token p in JSON at position 0
    at JSON.parse (<anonymous>)
    at vendors~main.0d71f224.js:67
    at Array.forEach (<anonymous>)
    at r.setStore (vendors~main.0d71f224.js:67)
    at new r (vendors~main.0d71f224.js:67)
    at Do (vendors~main.0d71f224.js:112)
    at ai (vendors~main.0d71f224.js:112)
    at di (vendors~main.0d71f224.js:112)
    at Wi (vendors~main.0d71f224.js:112)
    at Ki (vendors~main.0d71f224.js:112)
setStore @ vendors~main.0d71f224.js:67
main.5883766f.js:1 
```

**Comment by siddharthlatest** [comment]
@ivnnv We have released a new version (3.1.0) that should fix the above issues. I verified that the row selection feature works on the chrome extension.

![](https://i.imgur.com/9AjWJwS.png)

Closing this issue.
